tools/mozdevice/logging.js

```diff
@@ -107,6 +107,15 @@ Logging.prototype.error = function(tag, message) {
  */
 Logging.prototype.clear = function() {
   debug('Clearing');
+  if (this.device.properties['ro.build.version.sdk'] >= 21) {
+    var logging = this;
+    return new Command()
+      .append('date +"%m-%d %H:%M:%S.000"')
```

Comment:
Also, let's make this command a defacto constant at the top since it's used more than once:

```js
var NOW_COMMAND = 'date +"%m-%d %H:%M:%S.000"';
```